src/driver/postgres/PostgresQueryRunner.ts

```diff
@@ -1434,6 +1434,10 @@ export class PostgresQueryRunner extends BaseQueryRunner implements QueryRunner
             }
             return `("ns"."nspname" = '${schema}' AND "cl"."relname" = '${name}')`;
         }).join(" OR ");
+
+        const hasIsPartition = await this.hasPartitionedTables();
+        const isPartitionCondition = hasIsPartition ? ` AND "cl"."relispartition" = 'f'` : "";
```

Comment:
what if we wouldn't have `hasIsPartition` check here? according to this check it looks like we do need items with "AND "cl"."relispartition" = 't' ?

Reply:
@pleerock the column `relispartition` does not exist in PostgreSQL versions that have no support for partitioned tables (prior to version 10). This is why we need this check here.
Then, when using a version of PostgreSQL with partitioned tables, we need to ignore the partitions and only check the constraints on the "main" table ie: `"relispartition" = "f"` (which are not partitions).

Reply:
Maybe we should rename `hasPartitionedTables` to `hasSupportForPartitionedTables` to make it clearer?